resources/definitions/anycubic_kobra_plus.def.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": 2,
+    "name": "Anycubic Kobra Plus",
+    "inherits": "fdmprinter",
+    "metadata":
+    {
+        "visible": true,
+        "author": "Jordon Brooks",
+        "manufacturer": "Anycubic",
+        "file_formats": "text/x-gcode",
+        "has_machine_quality": true,
+        "has_materials": true,
+        "machine_extruder_trains": { "0": "anycubic_kobra_plus_extruder_0" },
+        "preferred_material": "generic_pla",
+        "preferred_quality_type": "pla",
```

Comment:
`pla` isn't the best name as a quality type, `standard`, `draft`, `normal` etc. are more inline with best practices.